modules/apps/object/object-service/src/main/java/com/liferay/object/model/impl/ObjectDefinitionImpl.java

```diff
@@ -66,6 +69,15 @@ public String getExtensionDBTableName() {
 		return getDBTableName() + "_x";
 	}
 
+	@Override
+	public Map<Locale, String> getLabelMap() {
+		Map<Locale, String> labelMap = super.getLabelMap();
+
+		labelMap.putIfAbsent(LocaleUtil.getDefault(), getLabelCurrentValue());
```

Comment:
We should not return a Label Map containing a fake option because many places use this method, so we would be taking a risk that this Label with this fake translation could be persisted.

The team is aware that there are objects that do not contain translations. In this case, we should display the default language information. So, the system should search for the default language key in the map to show in case there is no translation in the actual language. 

Since we know the translation limitation, the Object Definition API returns the property 'defaultLanguageId'. This property informs the default language ID of the object definition.

I saw that you worked on the code of the frontend-data-set. Do you think this behavior would be possible on the FDS side?

I think the FDS component could search for the default language id option if no translation is available.